package.json

```
@@ -4,17 +4,24 @@
   "description": "WaspBot (Discord)",
   "main": "src/bot.js",
   "author": "Martin Sosic (Wasp-lang)",
+  "scripts": {
+    "testAnalytics": "node ./src/analytics.js"
```

```suggestion
    "analytics": "node ./src/analytics.js",
    "bot": "node ./src/bot.js"
```
You might be running analytics locally not just to test them but to actually see what they return, so I would just call it `analytics`.
Also, if we have command for analytics, we can add one for `bot`, to be consistent.
We can then also update README to use `npm run bot` instead of `node ./src/bot.js`.